src/obserror/obserror2ioda.f90

```diff
@@ -33,17 +46,17 @@ program obserror2ioda
 call check( nf90_create(trim(filename_out), NF90_CLOBBER + NF90_NETCDF4,ncid_out))
 call check( nf90_def_dim(ncid_out, 'channels', nch_active,  nch_dimid) )
 call check( nf90_def_var(ncid_out, 'channels', NF90_INT, nch_dimid, varid_nch) )
-call check( nf90_def_var(ncid_out, "obserror_correlations", NF90_FLOAT, (/nch_dimid, nch_dimid/), varid_corr) )
+call check( nf90_def_var(ncid_out, trim(varid_name), NF90_FLOAT, (/nch_dimid, nch_dimid/), varid_corr) )
 call check( nf90_put_var(ncid_out, varid_nch, indxR) )
 call check( nf90_put_var(ncid_out, varid_corr, readR4) )
 call check( nf90_close(ncid_out) )
 
 contains
- subroutine check(status)
+  subroutine check(status)
     integer, intent ( in) :: status
     if(status /= nf90_noerr) then
-     print *, trim(nf90_strerror(status))
-     stop "Stopped"
+      print *, trim(nf90_strerror(status))
+      stop "Stopped"
```

Comment:
Would a
`call abort`
be a better way for any workflow to catch an abnormal exit?

Reply:
Sure, thanks for suggesting it. I used `stop` above to be consistent with what is the code already.